client/src/disabledGroups.spec.ts

```diff
@@ -16,7 +16,12 @@ describe("disabledRooms", () => {
     clock = fakeTimers.install({ now: now });
   });
 
-  function groupFor(config: { disabledBefore?: number; disabledAfter?: number; name: string }): Group {
+  function groupFor(config: {
+    disabledBefore?: number;
+    disabledAfter?: number;
+    joinableFrom?: number;
```

Comment:
This should be probably joinableAfter, to be consistent with disabledAfter

Reply:
Ack, done